src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -788,27 +906,35 @@ private Traverse GetTraverse(string direction, IVariableInformation node)
             return go;
         }
 
-        private class Node
+        /// <summary>
+        /// Traverse tree based on specified startIndex.
+        /// Then add wrappers for Natvis tree visualizations.
+        /// </summary>
+        /// <param name="root">Root of the tree</param>
+        /// <param name="goLeft">Left child of root</param>
```

Comment:
`goLeft` and `goRight` are not directly the child nodes. They are `Traverse` callbacks to handle grabbing the left child of the node passed into it. 

Same with `getValue`, it is a callback to obtain the value of the given node

Reply:
Okay will fix.